src/test/kotlin/model/LottoTest.kt

```diff
@@ -0,0 +1,65 @@
+package model
+
+import org.junit.jupiter.api.Assertions.assertDoesNotThrow
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.assertTrue
+
+class LottoTest {
+    @Test
+    fun `invalid purchase amount not divisible by unit`() {
```

Comment:
```suggestion
    fun `exception : invalid purchase amount - not divisible by unit`() {
```

This is just my opinion, but formatting test names consistently really helps with maintaining and writing new tests as a team.
The suggestion above is just an example. It’d be great for you and your pair to figure out what works best for you.